core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -497,8 +499,8 @@ struct SubviewExtents {
                                        const ViewDimension<DimArgs...>& dim,
                                        const std::pair<T, T>& val,
                                        Args... args) {
-    const size_t b = static_cast<size_t>(val.first);
-    const size_t e = static_cast<size_t>(val.second);
+    const size_t b = static_cast<size_t>(Kokkos::get<0>(val));
```

Comment:
why? What is better with this? I think we do not need to invoke a templated non member function to access a public variable in a non-generic setting ...

Reply:
Fair; I forgot to change the `std::pair` here to something more generic.  That was the whole point.